src/index.html

```diff
@@ -2,11 +2,394 @@
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <header class="header">
+      <a href="#home">
+        <img
+          src="images/logo.png"
+          alt="Moyo"
+          class="logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#laptops-computers"
+              class="nav__link "
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="cards">
+        <div class="product-card" data-qa="card">
+          <div class="product-logo"></div>
+
+          <h2 class="product-name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+          <div class="product-code">Product code: 195434</div>
+
+          <div class="reviews">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <div class="reviews__text">
+              Reviews: 5
+            </div>
+          </div>
+
+          <div class="price">
+            <div class="price__text">
+              Price:
+            </div>
+
+            <div class="price__number">
+              $2,199
+            </div>
+          </div>
+
+          <a
+            href="#"
+            class="link"
+            data-qa="hover"
+          >
+            <div class="buy" data-qa="card-hover">
+              BUY
+            </div>
+          </a>
```

Comment:
no needs to use `div` inside `a` tag. Style link
And use lowercased text at html (you can change this case at css)
```suggestion
          <a
            href="#"
            class="link"
            data-qa="hover"
            class="card__buy"
          >
            buy
          </a>
```

Reply:
if I take out the div my  Buy breaks down. When I've done this task before, it's accepted